### ISSUE TITLE
GithubRepoLoader not being included in build output

### ISSUE THREAD

**Issue by stevenschmatz** [hwchase17/langchainjs#423]
# Version

0.0.38

# Reproducible code example

```ts
import { GithubRepoLoader } from "langchain/document_loaders";
// note: GithubRepoLoader is not found

const loader = new GithubRepoLoader(
  "https://github.com/hwchase17/langchainjs",
  { branch: "main", recursive: false, unknown: "warn" }
);

(async () => {
  const docs = await loader.load();
  console.log({ docs });
})()

```

**Comment by nfcampos** [comment]
@stevenschmatz This has been merged but not released yet, will be released later today on 0.0.39

**Comment by nfcampos** [comment]
This has been published now on 0.0.39
